viewer/test/viewer-test-pptr.js

```diff
@@ -251,6 +265,25 @@ describe('Lighthouse Viewer', () => {
     });
   });
 
+  describe('Renders old reports', () => {
+    [
+      'lhr-3.0.0.json',
+      'lhr-4.3.0.json',
+      'lhr-5.0.0.json',
+      'lhr-6.0.0.json',
+      'lhr-8.5.0.json',
+    ].forEach((testFilename) => {
+      it(`[${testFilename}] should load with no errors`, async () => {
+        await viewerPage.goto(viewerUrl, {waitUntil: 'networkidle2', timeout: 30000});
+        const fileInput = await viewerPage.$('#hidden-file-input');
+        await fileInput.uploadFile(`${LH_ROOT}/report/test-assets/${testFilename}`);
+        await viewerPage.evaluate(() =>
+          document.addEventListener('lh-file-upload-test-ack', {once: true}));
```

Comment:
I'm a bit confused, shouldn't there be a callback function for this event listener?

Reply:
oopsies